apps/prairielearn/src/lib/server-jobs.ts

```diff
@@ -117,7 +118,7 @@ class ServerJobImpl implements ServerJob, ServerJobExecutor {
     args: string[] = [],
     options: ServerJobExecOptions,
   ): Promise<ServerJobResult> {
-    this.addToOutput(chalk.blueBright(`Command: ${file} ${args.join(' ')}\n`));
+    this.addToOutput(chalk.blueBright(`Command: ${shlex.join([file, ...args])}\n`));
```

Comment:
This is not directly related to the other code, but since shlex has been added might as well use it here... Note that the previous use would not be an error, since this is just logging, but now the logging will properly handle cases with e.g., spaces in arguments.